### ISSUE TITLE
Template: Focus not in meaningful order. - 1.3.2 Meaningful Sequence, 4.1.2 Name, Role, Value

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#81]
In the navigation, the search modal comes before the button to open the search modal, which means users may not find the search fields with keyboard. 

This can be fixed with moving focus to the modal as required by  [the ARIA Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/). 

This is a failure of 4.1.2 Name, Role, Value and 1.3.2 Meaningful Sequence

**Comment by syarwick23** [comment]
Issue fixed with new template!
